### DISCUSSION TITLE
I can't copy paste

### DISCUSSION THREAD

**Discussion by Offjuuzou** [Ylianst/MeshCentral#6485] [General]
Hello, do you know how to copy and paste with (CTRL +c and CTRL +v) from my computer to the remote station where I intervene remotely from meshcentral.
Because for me dont work 
Tu for help me

**Comment by aniruddhaadak80** [comment]
Hey @Offjuuzou ,

To enable clipboard sharing for copy-pasting between your computer and a remote machine in MeshCentral, try the following steps:

1. **Check Clipboard Permissions**: Ensure that clipboard sharing is enabled in the remote session settings within MeshCentral. Sometimes, clipboard access is restricted by default for security.

2. **Use the Correct Browser**: Some browsers have better support for clipboard functionality in remote sessions. MeshCentral often works best with Chrome or Firefox.

3. **MeshCentral Agent Update**: Make sure both the MeshCentral server and agents on the remote machines are updated, as older versions may lack full clipboard support.

4. **Keyboard Shortcuts**: If `CTRL+C` and `CTRL+V` don’t work, try right-clicking to copy and paste within the session, as this can sometimes bypass issues with shortcuts.

Hope it helps . 

Thank you .

**Reply by Offjuuzou** [reply]
clipboard access is restricted by default for security, what do you mean by that?

**Reply by Offjuuzou** [reply]
and have you already tried it from edge?

**Reply by si458** [reply]
To use copy+paste. U must use either Google Chrome or Microsoft edge, then before connecting to the remote pc, tick the copy clipboard under settings, then when u connect U can copy and paste, u can also go fullscreen and use ALL ur keyboard shortcuts too
